testing/src/test/java/org/motechproject/nms/testing/it/imi/ImiController_OBD_BundleIT.java

```diff
@@ -180,4 +180,28 @@ public void verifyFT200() throws IOException, InterruptedException {
         assertTrue(SimpleHttpClient.execHttpRequest(httpPost, HttpStatus.SC_BAD_REQUEST, expectedJsonResponse,
                 ImiTestHelper.ADMIN_USERNAME, ImiTestHelper.ADMIN_PASSWORD));
     }
+
+    /*
+    * NMS_FT_193 :To check "NotifyFileProcessedStatus" API is rejected in case the file could not
+    * be copied or the file is not available.
+    * NMS_FT_194 : To check "NotifyFileProcessedStatus" API is rejected in case there is an error in checksum
+    * NMS_FT_195 : To check "NotifyFileProcessedStatus" API is rejected in case there is an error in records check.
+    */
+    @Test
+    public void verifyFT193_194_195() throws IOException, InterruptedException {
```

Comment:
I don't agree with this shortcut of assuming that they are all the same. The enums passed are explicitly different for each of those test cases. The internal behavior might be the same that you wouldn't know that in the black box scenario. 

I agree that we just raise the same alert in each of those sub-scenarios but you are not truly testing the different scenarios here but just the output of multiple scenarios which happen to have the same result.

@srivastavaAbhi lets chat about this when you are in.